plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -333,6 +334,24 @@ public void testNumericAggregationPushdown()
         }
     }
 
+    @Test
+    public void testCountDistinctWithStringTypes()
+    {
+        if (!(hasBehavior(SUPPORTS_CREATE_TABLE) && hasBehavior(SUPPORTS_INSERT))) {
+            throw new SkipException("Unable to CREATE TABLE to test count distinct");
+        }
+        String tableName = getSession().getSchema().orElseThrow() + ".count_distinct_varchars";
```

Comment:
You don't need to qualify table name.

But please do randomize the name (`randomTableSuffix`)